appcli/cli_builder.py

```diff
@@ -138,15 +148,19 @@ def cli(
             generated_configuration_dir=configuration_dir.joinpath(".generated/conf"),
             app_configuration_file=configuration_dir.joinpath(f"{APP_NAME}.yml"),
             templates_dir=configuration_dir.joinpath("templates"),
-            project_name=f"{APP_NAME}-{environment}",
+            project_name=f"{APP_NAME}_{environment}",
             app_version=APP_VERSION,
             commands=default_commands,
         )
 
-        check_docker_socket()
-        check_valid_environment_variable_names([x[0] for x in additional_data_dir])
-        check_valid_environment_variable_names([x[0] for x in additional_env_var])
         relaunch_if_required(ctx)
```

Comment:
There are circumstances that this will fail, we might not have a docker socket. Why do we need to relaunch for the launcher?